Core/Core/Database/CoreData/Models/Submission.swift

```diff
@@ -199,7 +199,7 @@ extension Submission {
     public var subtitle: String? {
         guard let type = type else { return nil }
         switch type {
-        case .basic_lti_launch, .external_tool, .online_quiz:
+        case .basic_lti_launch, .external_tool, .online_quiz, .arc:
```

Comment:
I didn't realize they got their own submission type.

Reply:
It's not a submission type from the api. I created it and add it to the list of types if the course has arc.

Reply:
I think adding it to the submission types will be confusing in the long run. From multiple places you'd get the impression that it is coming from the api, and that this is the way we know it's an Arc submission.